pills/04-basics-of-language.xml

```diff
@@ -53,7 +53,8 @@
       <command>nix-env -i nix-repl</command>. The <literal>nix-repl</literal> syntax is slightly
       different to Nix syntax when it comes to assigning variables, but it shouldn't
       be confusing so long as you bear it in mind. I prefer to start with <literal>nix-repl</literal>
-      before cluttering your mind with more complex expressions.
+      before cluttering your mind with more complex expressions. If you are using a newer version of NixOS
+      while working through the pills you will want to use <command>nix repl</command> instead of <command>nix-repl</command>
```

Comment:
Is it to early to remove nix-repl altogether?

Reply:
The new cli is still experimental, but `nix-repl` is 1.11 only causing unexpected behaviour in some cases. I think it's fine in this case since it's just for interactive use.

Reply:
We should probably start with saying they want to use `nix repl` and then fall back to `nix-repl` if they're on old Nix. However, this is better than it was so I'll merge.